### ISSUE TITLE
Health-Check Potentially Causing container to silently exit

### ISSUE THREAD

**Issue by alandrees** [inspircd/inspircd-docker#100]
Full disclosure:  I built the image for armhf.

With that said, after getting the container working on my platform (building it using `docker build` seems to work fine), I found that the container was crashing every hour or so.

After the issue #99, I removed the health-check from the Dockerfile, and it's been up and running for 9 hours.

It seems it crashes after just about an hour and 10 minutes or so.

**Comment by SISheogorath** [comment]
Can you Start the container with the `--debug` flag?

**Comment by AeroSteveO** [comment]
I encountered this issue as well. I built the docker image on an RPI 3B and ran it using both docker-compose and docker swarm. This was using a custom configuration mounted to /inspircd/conf. I found that changing the healthcheck time to 15 seconds has given me a solid 24 hours of uptime. Previously i would have the container restart every hour or 2, with only rare occasions where it would run for 13 hours. Attached is a log file with the --debug flag enabled
[inspircd.debug.log](https://github.com/inspircd/inspircd-docker/files/4021501/inspircd.debug.log)
